content/en/agent/amazon_ecs/_index.md

```diff
@@ -0,0 +1,315 @@
+---
+title: Amazon ECS
+kind: documentation
+aliases:
+  - /integrations/amazon_ecs/
+further_reading:
+- link: "/agent/amazon_ecs/logs/"
+  tag: "Documentation"
+  text: "Collect your application logs"
+- link: "/agent/amazon_ecs/apm/"
+  tag: "Documentation"
+  text: "Collect your application traces"
+- link: "/agent/amazon_ecs/metrics/"
+  tag: "Documentation"
+  text: "Collect ECS metrics"
+---
+
+## Overview
+
+Amazon ECS on EC2 is a highly scalable, high-performance container management service for Docker containers running on EC2 instances.
```

Comment:
we should make it very clear to readers early on in the page, that for ECS on Fargate, they should follow this doc: https://docs.datadoghq.com/integrations/ecs_fargate/

Reply:
Started a new PR for these changes, resolved this here: https://github.com/DataDog/documentation/pull/8513